W-17979278-0321-acb-release

Crispy-Salesforce opened the pull request:
0321-acb-release

Crispy-Salesforce committed:
W-17875692-Studio-to-ACB-project-migration-CP

Crispy-Salesforce committed:
W-17875692-Update-page-layout-CP

Crispy-Salesforce committed:
W-17875692-Fix-typo-CP

W-17875692-Fix-typo-CP

Crispy-Salesforce committed:
W-17875692-Add-see-also-link-CP

W-17875692-Add-see-also-link-CP

Crispy-Salesforce committed:
W-17875692-Update-instructions-CP

Crispy-Salesforce committed:
W-17875692-Fix-See-Also-CP

W-17875692-Fix-See-Also-CP

Crispy-Salesforce committed:
W-17875692-Change-topic-location-in-TOC-CP

W-17875692-Change-topic-location-in-TOC-CP

Crispy-Salesforce committed:
W-17875692-Change-file-name-CP

W-17875692-Change-file-name-CP

Crispy-Salesforce committed:
W-17875692-Change-file-name-CP

W-17875692-Change-file-name-CP

Crispy-Salesforce committed:
W-17941905-review-AF-tut-with-DW-CP

Crispy-Salesforce committed:
W-17934499-Review-DW-issues-errors-CP

Crispy-Salesforce committed:
W-17934499-Address-VALE-feedback-CP

Crispy-Salesforce committed:
W-17951162-Review-DW-expressions-config-CP

Crispy-Salesforce committed:
W-17951162-Add-screenshot-UI-CP

Crispy-Salesforce committed:
W-17951162-Update-image-and-content-CP

Crispy-Salesforce committed:
W-17951162-Update-content-CP

Crispy-Salesforce committed:
W-17951162-Update-Config-DW-expressions-CP

Crispy-Salesforce committed:
W-17951162-Update-content-CP

Crispy-Salesforce committed:
W-17951162-Update-content-CP

Crispy-Salesforce committed:
W-17951162-Update-content-CP

Crispy-Salesforce committed:
W-17951162-Update-content-CP

Crispy-Salesforce committed:
W-17951162-Add-Transform-builder-UI-image-CP

Crispy-Salesforce committed:
W-17951162-Add-main-features-CP

Crispy-Salesforce committed:
W-17951162-Fix-format-CP

Crispy-Salesforce committed:
W-17951162-Update-content-CP

Crispy-Salesforce committed:
W-17917492-Address-Hanna-feedback-CP

W-17917492-Address-Hanna-feedback-CP

Crispy-Salesforce committed:
W-17917492-Address-Hanna-feedback-CP

W-17917492-Address-Hanna-feedback-CP

Crispy-Salesforce committed:
W-17917492-Address-Hanna-feedback-CP

W-17917492-Address-Hanna-feedback-CP

Crispy-Salesforce committed:
W-17917492-Address-Hanna-feedback-CP

W-17917492-Address-Hanna-feedback-CP

Crispy-Salesforce committed:
W-17951162-Address-image-issues-CP

Crispy-Salesforce committed:
W-17951162-Solve-nav-issues-CP

W-17951162-Solve-nav-issues-CP

Crispy-Salesforce committed:
W-17951162-Solve-nav-issues-CP

W-17951162-Solve-nav-issues-CP

Crispy-Salesforce committed:
W-17951162-Fix-issues-to-merge-CP

W-17951162-Fix-issues-to-merge-CP

Crispy-Salesforce committed:
W-17951162-Fix-conflict-issues-CP

Crispy-Salesforce committed:
Merge branch 'W-17951162-Review-DW-expressions-config-CP' of ssh://github.com/mulesoft/docs-code-builder into W-17951162-Review-DW-expressions-config-CP

Crispy-Salesforce committed:
W-17941905-Update-content-CP

W-17941905-Update-content-CP

Crispy-Salesforce committed:
W-17951162-Add-supported-use-cases-CP

Crispy-Salesforce committed:
W-17951162-Update-use-cases-table-CP

W-17951162-Update-use-cases-table-CP

Crispy-Salesforce committed:
Update modules/ROOT/pages/int-configure-dw-expressions.adoc

Co-authored-by: Tasha Moore <93338373+tasmoore@users.noreply.github.com>

Crispy-Salesforce committed:
Update modules/ROOT/pages/int-configure-dw-expressions.adoc

Co-authored-by: Tasha Moore <93338373+tasmoore@users.noreply.github.com>

Crispy-Salesforce committed:
Update modules/ROOT/pages/int-configure-dw-expressions.adoc

Co-authored-by: Tasha Moore <93338373+tasmoore@users.noreply.github.com>

Crispy-Salesforce committed:
Update modules/ROOT/pages/int-configure-dw-expressions.adoc

Co-authored-by: Tasha Moore <93338373+tasmoore@users.noreply.github.com>

Crispy-Salesforce committed:
Update modules/ROOT/pages/int-configure-dw-expressions.adoc

Co-authored-by: Tasha Moore <93338373+tasmoore@users.noreply.github.com>

Crispy-Salesforce committed:
Update modules/ROOT/pages/int-configure-dw-expressions.adoc

Co-authored-by: Tasha Moore <93338373+tasmoore@users.noreply.github.com>

Crispy-Salesforce committed:
Update modules/ROOT/pages/int-configure-dw-expressions.adoc

Co-authored-by: Tasha Moore <93338373+tasmoore@users.noreply.github.com>

Crispy-Salesforce committed:
Update modules/ROOT/pages/int-configure-dw-expressions.adoc

Co-authored-by: Tasha Moore <93338373+tasmoore@users.noreply.github.com>

Crispy-Salesforce committed:
Update modules/ROOT/pages/int-configure-dw-expressions.adoc

Co-authored-by: Tasha Moore <93338373+tasmoore@users.noreply.github.com>

Crispy-Salesforce committed:
W-17951162-Fix-typo-CP

W-17951162-Fix-typo-CP

Crispy-Salesforce committed:
Merge pull request #432 from mulesoft/W-17941905-Review-AF-tut-with-DW-CP

W-17941905-review-AF-tut-with-DW-CP

Crispy-Salesforce committed:
Merge pull request #434 from mulesoft/W-17934499-Review-DW-issues-errors-CP

W-17934499-Review-DW-issues-errors-CP

Crispy-Salesforce committed:
Merge pull request #435 from mulesoft/W-17951162-Review-DW-expressions-config-CP

W-17951162-Review-DW-expressions-config-CP

Crispy-Salesforce committed:
Merge pull request #429 from mulesoft/W-17875692-Studio-to-ACB-project-migration-CP

W-17875692-Studio-to-ACB-project-migration-CP

Crispy-Salesforce committed:
W-18030383-Address-pending-tasks-CP

Crispy-Salesforce committed:
Merge pull request #437 from mulesoft/W-18030383-Address-ACB-March-release-pending-tasks-CP

W-18030383-Address-pending-tasks-CP

valkyrie69 commented on modules/ROOT/pages/ai-topic-center.adoc:
```diff
@@ -0,0 +1,20 @@
+= MuleSoft for Agentforce: Topic Center Overview
```
This file shouldn't be in this PR.

valkyrie69 commented on modules/ROOT/pages/ai-enabling-api-project-topic-center.adoc:
```diff
@@ -1,17 +1,7 @@
 = Enabling an API Project for Topics and Agent Actions
-:page-aliases: ai-creating-topic-enabled-api-spec.adoc, ai-topic-center.adoc
+:page-aliases: ai-creating-topic-enabled-api-spec.adoc
```
This file shouldn't be in this PR.

valkyrie69 commented on modules/ROOT/nav.adoc:
```diff
@@ -41,10 +41,10 @@
-* xref:ai-enabling-api-project-topic-center.adoc[]
 
 // DESIGN
+* xref:ai-topic-center.adoc[]
```
This is incorrect. This change should not have been made. There are a couple of other changes made that are incorrect and overwrite what's currently published. 
The ai-enabling-api-project-topic-center doesn't belong in the Design section. It belongs where it is in the latest, published branch.

melissa-kulm-sfdc1 committed:
Delete modules/ROOT/nav.adoc

Crispy-Salesforce committed:
W-18040177-delete-wrong-commits-CP

W-18040177-delete-wrong-commits-CP

Crispy-Salesforce committed:
W-18040177-delete-wrong-commits-CP

W-18040177-delete-wrong-commits-CP

Crispy-Salesforce committed:
W-18040177-add-nav-CP

Crispy-Salesforce committed:
Merge pull request #440 from mulesoft/W-18040177-delete-wrong-commits-CP

W-18040177-delete-wrong-commits-CP

valkyrie69 committed:
Adding updated content

valkyrie69 committed:
W-17990075 Publish to Exchange feature

Crispy peer review feedback

valkyrie69 committed:
Adding content

valkyrie69 committed:
Adding content and anchors

valkyrie69 committed:
Adding content

valkyrie69 committed:
Adding content

valkyrie69 committed:
fixes

valkyrie69 committed:
formatting

valkyrie69 committed:
Update des-publish-api-spec-to-exchange.adoc

valkyrie69 committed:
Review feedback

valkyrie69 committed:
Review feedback

valkyrie69 committed:
updating image

valkyrie69 committed:
Edits

valkyrie69 committed:
Small addition

melissa-kulm-sfdc1 committed:
W-17875303 add connectors-mk

Add Agentforce and Einstein AI connectors

valkyrie69 committed:
fix

melissa-kulm-sfdc1 committed:
Update modules/ROOT/pages/_partials/acb-component-info.adoc

Co-authored-by: sathya0 <78386709+sathya0@users.noreply.github.com>

melissa-kulm-sfdc1 committed:
Update modules/ROOT/pages/_partials/acb-component-info.adoc

Co-authored-by: sathya0 <78386709+sathya0@users.noreply.github.com>

valkyrie69 committed:
Added a couple of things

melissa-kulm-sfdc1 committed:
Reorder Einstein AI Connector

Alphabetical order

melissa-kulm-sfdc1 committed:
Merge pull request #443 from mulesoft/W-17875303-add-Agentforce-and-Einstein-AI-connectors-mk

W-17875303 add connectors-mk

valkyrie69 committed:
Update ai-enabling-api-project-topic-center.adoc

valkyrie69 committed:
Merge pull request #441 from mulesoft/W-17990075-publish-exchange-vh

W-17990075 Publish to Exchange feature

melissa-kulm-sfdc1 committed:
Updated for conversation support enhancements and replaced terms

Replaced Einstein with Agentforce and prompt with message, along with conversation support updates

melissa-kulm-sfdc1 committed:
Resized screenshots, updated links and text

Edits for screenshots and terminology

melissa-kulm-sfdc1 committed:
reduced screenshot size

reduced screenshot size

melissa-kulm-sfdc1 committed:
reduced screenshot size

reduced screenshot size

melissa-kulm-sfdc1 committed:
added PM feedback

added PM feedback

melissa-kulm-sfdc1 committed:
Update modules/ROOT/pages/int-ai-create-integrations.adoc

Co-authored-by: Cristian Pose <101070178+Crispy-Salesforce@users.noreply.github.com>

melissa-kulm-sfdc1 committed:
Update modules/ROOT/pages/int-ai-create-integrations.adoc

Co-authored-by: Cristian Pose <101070178+Crispy-Salesforce@users.noreply.github.com>

melissa-kulm-sfdc1 committed:
Update modules/ROOT/pages/int-ai-create-integrations.adoc

Co-authored-by: Cristian Pose <101070178+Crispy-Salesforce@users.noreply.github.com>

melissa-kulm-sfdc1 committed:
Update modules/ROOT/pages/int-ai-create-integrations.adoc

Co-authored-by: Cristian Pose <101070178+Crispy-Salesforce@users.noreply.github.com>

melissa-kulm-sfdc1 committed:
Update modules/ROOT/pages/int-ai-create-integrations.adoc

Co-authored-by: Cristian Pose <101070178+Crispy-Salesforce@users.noreply.github.com>

melissa-kulm-sfdc1 committed:
Update modules/ROOT/pages/int-ai-developing-integrations.adoc

Co-authored-by: Cristian Pose <101070178+Crispy-Salesforce@users.noreply.github.com>

melissa-kulm-sfdc1 committed:
Update modules/ROOT/pages/int-ai-developing-integrations.adoc

Co-authored-by: Cristian Pose <101070178+Crispy-Salesforce@users.noreply.github.com>

melissa-kulm-sfdc1 committed:
Update modules/ROOT/pages/troubleshoot-generative-ai.adoc

Co-authored-by: Cristian Pose <101070178+Crispy-Salesforce@users.noreply.github.com>

melissa-kulm-sfdc1 committed:
Update modules/ROOT/pages/troubleshoot-generative-ai.adoc

Co-authored-by: Cristian Pose <101070178+Crispy-Salesforce@users.noreply.github.com>

melissa-kulm-sfdc1 committed:
added padding to screenshots

added padding to screenshots

melissa-kulm-sfdc1 committed:
Merge branch 'W-18068528-add-conversation-support-enhancements-mk' of ssh://github.com/mulesoft/docs-code-builder into W-18068528-add-conversation-support-enhancements-mk

melissa-kulm-sfdc1 committed:
Merge pull request #445 from mulesoft/W-18068528-add-conversation-support-enhancements-mk

W-18068528 add conversation support enhancements mk

valkyrie69 committed:
Experimenting

valkyrie69 committed:
Update des-publish-api-spec-to-exchange.adoc

valkyrie69 committed:
fixing image (hopefully)

valkyrie69 committed:
If at first you don't succeed...

hannanelson committed:
removed extra spacer rectangle

Crispy-Salesforce committed:
W-18085797-Fix-bugs-in-nav-CP

Crispy-Salesforce committed:
Merge pull request #446 from mulesoft/W-18085797-Fix-bugs-before-merging-CP

W-18085797-Fix-bugs-in-nav-CP
